gitpod-io/website
# fix related url for in-product changelog

## Description


See also [internal chat](https://gitpod.slack.com/archives/C022HAT0H5Z/p1659604767048069?thread_ts=1659529344.767839&cid=C022HAT0H5Z)

## Related Issue(s)

Fixes #

## How to test

Check if latest changelog URLs are all absolute URL

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2542"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2542"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
